modules/packages/Python.chpl

```diff
@@ -1049,20 +1054,74 @@ module Python {
     /*
       Converts an array to a python set
     */
-    // proc toSet(arr): PyObjectPtr throws {
-    // TODO
-    // }
+    proc toSet(arr): PyObjectPtr throws
+      where isArrayType(arr.type) &&
+            arr.rank == 1 && arr.isDefaultRectangular() {
+      var pySet = PySet_New(nil);
+      this.checkException();
+      for i in 0..<arr.size {
+        const idx = arr.domain.orderToIndex(i);
+        PySet_Add(pySet, toPython(arr[idx]));
+        this.checkException();
+      }
```

Comment:
This loop looks fine, but since the index isn't really used except to get elements from the array, this could also be
```
for a in arr {
  PySet_Add(pySet, toPython(a));
  this.checkException();
}
```